src/EnergyPlus/Data/EnergyPlusData.hh

```diff
@@ -1,585 +1,587 @@
-    std::unique_ptr<TARCOGCommonData> dataTARCOGCommon;
-    std::unique_ptr<TARCOGOutputData> dataTARCOGOutputs;
-    std::unique_ptr<ThermalChimneysData> dataThermalChimneys;
-    std::unique_ptr<ThermalComfortsData> dataThermalComforts;
-    std::unique_ptr<ThermalISO15099CalcData> dataThermalISO15099Calc;
-    std::unique_ptr<TARCOGGasses90Data> dataTARCOGGasses90;
-    std::unique_ptr<TARCOGMainData> dataTARCOGMain;
-    std::unique_ptr<TarcogShadingData> dataTarcogShading;
-    std::unique_ptr<TranspiredCollectorData> dataTranspiredCollector;
-    std::unique_ptr<UCSDSharedData> dataUCSDShared;
-    std::unique_ptr<UFADManagerData> dataUFADManager;
-    std::unique_ptr<UnitHeatersData> dataUnitHeaters;
-    std::unique_ptr<UnitVentilatorsData> dataUnitVentilators;
-    std::unique_ptr<UnitarySystemsData> dataUnitarySystems;
-    std::unique_ptr<UserDefinedComponentsData> dataUserDefinedComponents;
-    std::unique_ptr<UtilityRoutinesData> dataUtilityRoutines;
-    std::unique_ptr<VariableSpeedCoilsData> dataVariableSpeedCoils;
-    std::unique_ptr<VectorsData> dataVectors;
-    std::unique_ptr<VentilatedSlabData> dataVentilatedSlab;
-    std::unique_ptr<ViewFactorInfoData> dataViewFactor;
-    std::unique_ptr<WaterCoilsData> dataWaterCoils;
-    std::unique_ptr<WaterManagerData> dataWaterManager;
-    std::unique_ptr<WaterThermalTanksData> dataWaterThermalTanks;
-    std::unique_ptr<WaterToAirHeatPumpData> dataWaterToAirHeatPump;
-    std::unique_ptr<WaterToAirHeatPumpSimpleData> dataWaterToAirHeatPumpSimple;
-    std::unique_ptr<WaterUseData> dataWaterUse;
-    std::unique_ptr<WeatherManagerData> dataWeatherManager;
-    std::unique_ptr<WindTurbineData> dataWindTurbine;
-    std::unique_ptr<WindowACData> dataWindowAC;
-    std::unique_ptr<WindowComplexManagerData> dataWindowComplexManager;
-    std::unique_ptr<WindowEquivLayerData> dataWindowEquivLayer;
-    std::unique_ptr<WindowEquivalentLayerData> dataWindowEquivalentLayer;
-    std::unique_ptr<WindowManagerData> dataWindowManager;
-    std::unique_ptr<ZoneAirLoopEquipmentManagerData> dataZoneAirLoopEquipmentManager;
-    std::unique_ptr<ZoneContaminantPredictorCorrectorData> dataZoneContaminantPredictorCorrector;
-    std::unique_ptr<ZoneDehumidifierData> dataZoneDehumidifier;
-    std::unique_ptr<ZoneEquipmentManagerData> dataZoneEquipmentManager;
-    std::unique_ptr<ZonePlenumData> dataZonePlenum;
-    std::unique_ptr<ZoneTempPredictorCorrectorData> dataZoneTempPredictorCorrector;
-
-    EnergyPlusData();
-    ~EnergyPlusData();
-
-    // Cannot safely copy or delete this until we eradicate all remaining
-    // calls to IOFiles::getSingleton and IOFiles::setSingleton
-    EnergyPlusData(const EnergyPlusData &) = delete;
-    EnergyPlusData(EnergyPlusData &&) = delete;
-
-    void clear_state() override;
-};
-
-} // namespace EnergyPlus
-#endif
+// EnergyPlus, Copyright (c) 1996-2022, The Board of Trustees of the University of Illinois,
```

Comment:
Why is this file showing all the lines changed?  Did something break with line endings?

Reply:
(please address this one)

Reply:
I noticed this too. It actually happened after pulling in develop. I had to manually merge conflicts that really shouldn't have been required.

Reply:
I tried to fix this yesterday but it hasn't stuck. In one file comparison tool, I noticed that line endings were `0x 0D 0A` on our manually merged copy, and `0x 0A `on develop, but on another tool I see `CRLF` on both. Yesterday's check in should have made them consistently 0x 0A, but it's still showing as a diff...

Reply:
Do you want me to take a shot at fixing the issue?  I'm happy to try if I have write access to this branch.

Reply:
If it helps, this popped up when I merged in the change that removed the PackagedTerminalAirConditioner (or HeatPump?) from the list.

Reply:
Yeah, it's showing up as CRLF on my develop checkout as well.  But in this branch the line endings are normalized back.  This is a good correction so just thanks and carry on!